migration-guide.md

```diff
@@ -0,0 +1,5 @@
+# Migration Guide
+
+## v1 to v3
+
+The primary change between version 1 and 3 of this action is that v1 installs the C# CLI (`octo`) and v3 installs the [Go CLI](https://g.octopushq.com/ExternalToolOctopus) (`octopus`).
```

Comment:
this link is not currently working, is this expected?

Reply:
it won't until the dependent PR merges :)